lib/entrypoint.sh

```diff
@@ -8,8 +8,8 @@ cp /git-grep-problem-matcher.json "$matcher_path"
 
 echo "::add-matcher::git-grep-problem-matcher.json"
 
-tag="FIXME"
-result=$(git grep --no-color -n -e "${tag}:")
+tag=${INPUT_TERMS:=FIXME}
+result=$(git grep --no-color --ignore-case --line-number --extended-regexp -e "(${tag})+(:)" :^.github)
```

Comment:
since we are using `docker` runner here, I don't think we can use `case-sensitive`, due to naming restriction not in github actions but in shell itself

> A name is a sequence of letters, digits, and underscores that begins with an underscore or a letter.

github actions in general, especially the official one are built in javascript which gave them more options and ability to control what the name of the input passed or the type of it.

The only option I found for this, other than rewrite everything in JS, is to pass the variable as an argument to the Docker image.

Reply:
Okay then, I'll remove it and wait until approving this one to submit the second PR 👍

Reply:
@bbugh can you have another look now?